server/app/views/admin/programs/ProgramIndexView.java

```diff
@@ -47,14 +51,14 @@ public final class ProgramIndexView extends BaseHtmlView {
   private final AdminLayout layout;
   private final String baseUrl;
   private final DateConverter dateConverter;
-  private final boolean statusTrackingEnabled;
+  private final Provider<Boolean> statusTrackingEnabled;
 
   @Inject
   public ProgramIndexView(
       AdminLayoutFactory layoutFactory,
       Config config,
       DateConverter dateConverter,
-      @ApplicationStatusTrackingEnabled boolean statusTrackingEnabled) {
+      @ApplicationStatusTrackingEnabled Provider<Boolean> statusTrackingEnabled) {
```

Comment:
There are two other places that inject this value that also need to be updated:
* AdminProgramStatusesController
* ProgramTranslationView

Reply:
done!!